chef/cookbooks/updater/recipes/default.rb

```diff
@@ -17,6 +17,21 @@
 # limitations under the License.
 #
 
+include CrowbarPacemaker::MaintenanceModeHelpers
+
+def set_maintenance_mode
```

Comment:
I just noticed this function is copy-n-pasted from https://github.com/crowbar/crowbar-ha/blob/master/chef/cookbooks/crowbar-pacemaker/providers/service.rb#L221 and IMHO that's not right because

1. it's a DRY violation, and
2. Pacemaker-specific code must live in `crowbar-ha` not this repo.